packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -111,22 +111,23 @@ export class Loan {
   /**
    * List collateral tokens.
    *
+   * @param {ListCollateralTokens} [collateralToken = {}]
+   * @param {number} [collateralToken.height = CurrentBlockheight] Valid at specified height
+   * @param {boolean} [collateralToken.all] True = All transactions, false =  Activated transactions
```

Comment:
one more filtered by token

Reply:
I am unsure when they will fix this. They promise me that listCollateralTokens won't accept token as parameter anymore.

Reply:
Only getCollateralToken will accept token as parameter

Reply:
weird.. because there has token filter.. will that be removed??

Reply:
I shall text them again

Reply:
Okay, He just told me that he would fix this by today. He was busy with something else.